blogs/godsakani.json

```diff
@@ -0,0 +1,16 @@
+{
+    "posts": [
+      {
+      "Title": "14 easy and cheap ways to block neighbors view",
+      "Description": "Blogs tells us 14 ways to create privacy in our homes",
+      "URL": "https://skintdad.co.uk/cheap-ways-to-block-neighbors-view/",
+      "Author": " NAOMI WILLIS"
+    },
+      {
+      "Title": "How to Promote Local Small Businesses – 5 Proven Effective Methods!",
+      "Description": "This blog tells us how to better manage our savings",
+      "Link": "https://article.advertiseera.com/how-to-promote-local-small-businesses-5-proven-effective-methods/",
+      "Author": "STAFFUSER"
+    }

```

Comment:
Please Is this tech related?
Just to be clear

Reply:
ok sir